antora-ui-camel/src/partials/nav.hbs

```diff
@@ -1,5 +1,9 @@
 <div class="nav-container"{{#if page.component}} data-component="{{page.component.name}}" data-version="{{page.version}}"{{/if}}>
+  {{#if (eq page.component.name 'components')}}
+  <aside class="nav nav-components">
+  {{else}}
   <aside class="nav">
+  {{/if}}
```

Comment:
I think this could be simpler, all `aside.nav` should cater to the same positioning, not sure why components would be different, perhaps the parent of this aside (`.nav-container`) should be positioned instead.